### DISCUSSION TITLE
How can I use Start Date and End Date in Perform Query?

### DISCUSSION THREAD

**Discussion by sanitghosh** [freekmurze/laravel-analytics#427] [Q&A]
Hello,

I have two questions here below -
1. How can I get only today analytics data by perform query method?
2. How can use Start and End day parameter in perform query method?

Thanks in advance.

**Comment by sanjaydeveloper15** [comment]
Hi 👋
I am getting login required error

I am using json file and view ID both
As per documentation there is no option given where we need to put oauth access token

So do you have any idea regarding it ?
